impl/src/main/kotlin/com/android/declarative/internal/DependencyProcessor.kt

```diff
@@ -54,17 +56,23 @@ class DependencyProcessor(
                         )
                     }
                 }
-                is FilesDependencyInfo -> {
+                is Files -> {
                     addFilesDependency(dependency)
                 }
-                is MavenDependencyInfo -> {
+                is Maven -> {
                     addLibraryDependency(dependency)
                 }
+                is ExtensionFunction -> {
+                    throw RuntimeException("Not Supported yet !")
+                }
+                is Alias -> {
```

Comment:
I'm thinking that it's rather reference to alias than declaration to alias

Reply:
not sure what you mean ? rename alias to reference ?

Reply:
Yes, but it requires updating model in studio so probably in another CL